src/pages/studyView/charts/scatterPlot/StudyViewDensityScatterPlot.tsx

```diff
@@ -14,7 +14,7 @@ import {
 } from 'cbioportal-frontend-commons';
 import { computed, makeObservable, observable } from 'mobx';
 import autobind from 'autobind-decorator';
-import { tickFormatNumeral } from '../../../../shared/components/plots/TickUtils';
+import { tickFormatNumeral } from 'cbioportal-frontend-commons/src/lib/TickUtils';
```

Comment:
this is not right way to import from package.  you have to expose it via index.tsx so that you can just do 
```
import blah from 'cbioportal-frontend-commons'
```

.   there's a test that catches this.

Reply:
ah yeah, I think the refactoring function messed that up